app/assets/stylesheets/base/typography.scss

```diff
@@ -1,4 +1,15 @@
 @import '../mixins';
+@import '../config/import';
+
+:root {
+  --content-rythm: var(--su-2);
```

Comment:
If you wanted to make another change in this PR, you can update the typo global in this variable from `content-rythm` to `content-rhythm` :P :P English is weird :P

Reply:
oh lol damn it